setup.py

```diff
@@ -22,11 +22,12 @@
     install_requires=[
        # -*- Extra requirements: -*-
     ],
-    entry_points=
-    """
+    entry_points='''
         [ckan.plugins]
-        # Add plugins here, eg
-        # myplugin=ckanext.geonetwork:PluginClass
+        geonetwork = ckanext.geonetwork.plugin:GeoNetworkPlugin
```

Comment:
what is this new plugin for?

Reply:
I did that for testing purpose and I forgot to remove that,
Thanks for pointing out, I removed those changes.

Reply:
Hello @etj 
When I remove plugin , 
I got this error,
there is no schema supplied error.


![image](https://user-images.githubusercontent.com/104504055/170480500-010adf10-a66f-4902-85d7-8e2f7743eabe.png)

Reply:
It should not be related to the geonetwork plugin, and it is not related to this extension.
Check how you set the "licenses_group_url" entry in your .in file .